en/SUMMARY.md

```diff
@@ -13,8 +13,8 @@
 1. [Mobile phones](/pages/mobile-phones.md#mobile-phone)
   1. [Pre-paid options](/pages/mobile-phones.md#other-options)
 1. [Opening a bank account](/pages/bank-account.md#opening-a-bank-account)
-  1. [Deutsche Bank](/pages/bank-account.md#deutsche-bank)
-  1. [N26](/pages/bank-account.md#n26)
+  1. [Normal Banks](/pages/bank-account.md#normal-banks)
+  1. [Only Online Banks](/pages/bank-account.md#only-online-banks)
```

Comment:
Can you change from "Normal Banks" / "Only Online Banks" to "Traditional Banks" / "Online banks"?

Reply:
Absolutely, sounds much better